multicluster/cmd/gateways.go

```diff
@@ -31,10 +34,10 @@ type (
 	}
 
 	gatewayStatus struct {
-		clusterName      string
-		alive            bool
-		numberOfServices int
-		latency          uint64
+		ClusterName      string `json:"clusterName"`
```

Comment:
I guess the visibility changed here because we want to marshal it? 🤔

Reply:
Yes right, @mateiidavid 
Without this, we are not able to marshal it to JSON fields.